core/trino-main/src/main/java/io/trino/execution/StageStats.java

```diff
@@ -626,6 +626,14 @@ public List<OperatorStats> getOperatorSummaries()
         return operatorSummaries;
```

Comment:
commit message too long. Try to fix in 70 chars and add longer explanation as descriptions